### DISCUSSION TITLE
[JS] Providing the `union-default-graph` query parameter

### DISCUSSION THREAD

**Discussion by ludovicm67** [oxigraph/oxigraph#734] [Q&A] [answered]
Hi all,

Same as #632, I'm wondering how to provide that `union-default-graph` query parameter using the JavaScript version ; or to be more precise, the Node.JS version in my case.

Is it an option that is available? Or should I open a "Feature request" issue for this?

**Comment by Tpt** [comment, marked as answer]
Hi! Thank you for using Oxigraph. Sadly this option is not exposed by the JS bindings yet (even if it should). A feature request would be very welcome! Thank you!

**Reply by ludovicm67** [reply]
Great, thank you for your answer! I opened this issue as feature request: https://github.com/oxigraph/oxigraph/issues/735.
